Libraries/LibJS/Bytecode/Interpreter.h

```diff
@@ -106,6 +107,63 @@ class JS_API Interpreter {
     Span<Value> m_registers_and_constants_and_locals_arguments;
     ExecutionContext* m_running_execution_context { nullptr };
     ReadonlySpan<Utf16FlyString> m_identifier_table;
+
+    template<typename OP>
+    friend void handle(Interpreter& interpreter, u8 const* bytecode, size_t& program_counter);
+    template<typename OP>
+    friend void handle_generic(Interpreter& interpreter, u8 const* bytecode, size_t& program_counter);
+    template<typename OP>
+    friend void handle_jump(Interpreter& interpreter, u8 const* bytecode, size_t& program_counter);
+#define ComparisonOpImpl(op, op_snake_case, numeric_operator) \
+    friend void handle_Jump##op##_impl(Interpreter& interpreter, u8 const* bytecode, size_t& program_counter);
+    JS_ENUMERATE_COMPARISON_OPS(ComparisonOpImpl)
+#undef ComparisonOpImpl
+
+    // clang-format off
+    // Clang keeps increasing the indent, as the macros dont have trailing commas
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wc99-designator"
+    // The [index] = value, array declaration style is a C99 feature, but both GCC and Clang support it in C++,
+    // but warn about it.
+    // This is mainly used to allow for allowing us to declare the values out of order,
+    // with macros generating most of the entries.
```

Comment:
You can do this without the silly feature, just generate it in a consteval iife like-